keanu-project/src/main/java/io/improbable/keanu/model/ModelFitter.java

```diff
@@ -2,4 +2,5 @@
 
 public interface ModelFitter<INPUT, OUTPUT> {
     void fit(INPUT input, OUTPUT output);
```

Comment:
`fit` no longer needs the `input` and `output` parameters, now that you've decoupled it from `observe`